docs/src/guides/working-with-projects.md

```diff
@@ -0,0 +1,172 @@
+---
+title: Working with Projects
+navTitle: Working with Projects
+description: Working with Hub Projects
+order: 12
+group: 2-concepts
+---
+
+## Working with Projects
+
+### In a Component
+
+If you have a component that you want to fetch and display a project using either
+the project's ID or slug
+
+```html
+<arcgis-hub-project-view identifier="dcdev-smith-st-2024" />
+```
+
+In the component, import the `getProject` function from `@esri/hub-common`, and use that
+in the lifecycle hooks to load the project. Directly importing the helper function ensures that the build will produce the smallest payload as compared to importing `HubProjectStore`.
+
+```ts
+import {getProject} from "@esri/hub-common";
+
+async componentWillLoad() {
+  if (!this.project && this.identifier && this.context) {
+    this.project = await getProject(this.identifier, this.context.requestOptions);
+  }
+}
+```
+
+### In an Application
+
+Assuming the following routes in an app...
+
+```js
+//...snip...
+this.route("projects", function () {
+  this.route("project", { path: "/:project_id" }, function () {
+    this.route("edit");
+  });
+  this.route("edit", function () {
+    this.route("new");
+  });
+});
+//...snip...
+```
+
+#### Create a Project
+
+To create a project in the `/projects/edit/new` controller...
+
+```js
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
+import { action } from "@ember/object";
+
+export default class projectsEditNewController extends Controller {
+  @service appSettings;
+
+  @action
+  createProject(project) {
+    const store = HubProjects.create(appSettings.contextManager);
```

Comment:
like otherfiles,use`projectStore` so it's clearer with `projecStore.create(...)`